lib/rubocop/cop/rspec/factory_bot/create_list.rb

```diff
@@ -30,9 +30,10 @@ class CreateList < Cop
           MSG_CREATE_LIST = 'Prefer create_list.'.freeze
           MSG_N_TIMES = 'Prefer %<number>s.times.'.freeze
 
-          def_node_matcher :n_times_block?, <<-PATTERN
+          def_node_matcher :n_times_block_without_arg?, <<-PATTERN
             (block
               (send (int _) :times)
+              (args)
```

Comment:
What does `(args)` mean here? Without arg?

Reply:
empty arguments list, e.g. no `| |` in the block

Reply:
Ah, of course. It would be e.g. `(args _)` if there were any arguments.